wildfly-openssl.yaml

```diff
@@ -49,7 +49,8 @@ pipeline:
 
 update:
   enabled: false
```

Comment:
We can flag updates as manual and leave them enabled so that automation creates a PR when the primary repo is updated

Reply:
Hi RJ, thanks for the quick review! I think the manual is still set to true or am I missing something https://github.com/wolfi-dev/os/pull/25949/files#diff-1d11820019c3e3f5ed1a82c8c8f1d4b3e2ee95659545023a21dad768a1022912R53

Reply:
Oh you mean both update and manual set to true/enabled I think

Reply:
Done @EyeCantCU, thanks so much for the review!